Claudia/src/main/java/FirstTask/PrimitiveData.java

```diff
@@ -0,0 +1,15 @@
+package FirstTask;
+
+public class PrimitiveData {
+    public static void main(String[] args) {
+
+        double radius;
+        double area;
+
+        radius = 5;
```

Comment:
Why you didn't initialize this variable on line 6?

Reply:
Done